### PR TITLE
RHEL 6 timeout has no -k option

### PR DESCRIPTION
**Pull request by gravitypriest**
uhhhhh whoops

### CODE REVIEW

File: insights/client/insights_spec.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def get_output(self):
 
         # all commands should timeout after a long interval so the client does not hang
         # prepend native nix 'timeout' implementation
-        timeout_command = 'timeout -k 5 %s %s' % (
+        timeout_command = 'timeout -s KILL %s %s' % (
```
Comment (bfahr) [inline insights/client/insights_spec.py]: @gravitypriest should it be `timeout -s KILL 5 %s %s`?

**Comment by gravitypriest** [comment]
The 5 is an argument to -k, i.e., if it doesn't respond to the timeout
signal, send KILL after 5 seconds

**Comment by bfahr** [comment]
On RHEL6 this is what I get:
```
$ timeout -s KILL sleep 10
timeout: invalid time interval `sleep'
Try `timeout --help' for more information.

$ timeout -s KILL 5 sleep 10
Killed
```

**Comment by gravitypriest** [comment]
In this case the command being timed out would be `sleep 10`, which goes into the second `%s` in the command string.

Timeout interval goes into the first `%s`, which is separate from the original `-k 5`.

Thus the full thing would be `timeout -s KILL 5 sleep 10`, as you saw.

**Comment by bfahr** [comment]
@gravitypriest makes sense to me, thanks for confirming.

**Pull request merged**